.github/workflows/release.yml

```diff
@@ -12,25 +12,16 @@ jobs:
     if: ${{ startsWith(github.ref, 'refs/tags/v') || startsWith(github.ref, 'refs/heads/release/v') }}
     runs-on: ubuntu-20.04
     steps:
-      - name: '❄ Wait for Hydra build'
-        uses: rvl/hydra-build-products-action@master
-        id: hydra
-        with:
-          hydra: 'https://hydra.iohk.io'
-          jobs: 'linux.musl.cardano-wallet-linux64 macos.intel.cardano-wallet-macos-intel linux.windows.cardano-wallet-win64'
-
-      - name: '🍒 Fetch release files'
-        run: |
-          wget ${{ steps.hydra.outputs.buildProducts }}
-
+      # Creates an empty release draft
+      # TODO: get artifacts from Buildkite to be attached to the draft
```

Comment:
Is there a JIRA task that corresponds to this TODO? If yes, it would be nice to mention it here in order not to forget to address this todo later.

Reply:
Yes, Jira ticket is one line below.